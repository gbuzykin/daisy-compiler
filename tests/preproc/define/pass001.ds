#define MACRO ((1) + (2))

const a = MACRO;