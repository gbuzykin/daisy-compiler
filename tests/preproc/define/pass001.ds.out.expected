./preproc/define/pass001.ds:3:1: debug: token
 3 | const a = MACRO;
   | ^~~~~
./preproc/define/pass001.ds:3:7: debug: id: a
 3 | const a = MACRO;
   |       ^
./preproc/define/pass001.ds:3:9: debug: token
 3 | const a = MACRO;
   |         ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:15: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |               ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:16: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                ^
./preproc/define/pass001.ds:3:11: debug: integer number: 1
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:17: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                 ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:18: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                  ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:20: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                    ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:22: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                      ^
./preproc/define/pass001.ds:3:11: debug: integer number: 2
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:23: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                       ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:24: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                        ^
./preproc/define/pass001.ds:3:11: debug: token
 3 | const a = MACRO;
   |           ^~~~~
./preproc/define/pass001.ds:1:25: note: expanded from macro `MACRO`
 1 | #define MACRO ((1) + (2))
   |                         ^
./preproc/define/pass001.ds:3:7: debug: defining constant `a`
 3 | const a = MACRO;
   |       ^
./preproc/define/pass001.ds:3:16: debug: token
 3 | const a = MACRO;
   |                ^
./preproc/define/pass001.ds: info: warnings 0, errors 0
